Azure/azure-cli
# {CosmosDB} Add test

**Related command**

- az cosmosdb list-connection-strings **OR** az cosmosdb keys list --type connection-strings
- az cosmosdb locations list **OR** az cosmosdb locations show --location {locationName}

**Description**
This PR contains the necessary changes to support the 2023-03-15 RP API version. The following changes have been made:
- Added test to validate additional fields in list connection strings response
- Added test to validate additional fields added in location API response

**Testing Guide**
For the list connection strings response update, run one of the following to see the updated response which include keyKind and type:
- az cosmosdb list-connection-strings --name {db-name} -g {resource-group} **OR** az cosmosdb keys list --type connection-strings --name  {db-name} -g {resource-group}

For the Location API changes run one of the following command to see updated response:
- az cosmosdb locations list
- az cosmosdb locations show --location {locationName}

**History Notes**


---

This checklist is used to make sure that common guidelines for a pull request are followed.
